dockerfiles/che/Dockerfile.centos

```diff
@@ -24,17 +24,10 @@ FROM registry.centos.org/centos/centos:latest
 
 ENV LANG=C.UTF-8 \
     JAVA_HOME=/usr/lib/jvm/jre-1.8.0 \
-    PATH=${PATH}:${JAVA_HOME}/bin \
-    CHE_HOME=/home/user/che \
-    DOCKER_VERSION=1.6.0 \
-    DOCKER_BUCKET=get.docker.com \
-    CHE_IN_CONTAINER=true
+    PATH=${PATH}:${JAVA_HOME}/bin
 
 RUN yum -y update && \
-    yum -y install openssl java sudo && \
-    curl -sSL "https://${DOCKER_BUCKET}/builds/Linux/x86_64/docker-${DOCKER_VERSION}" -o /usr/bin/docker && \
-    chmod +x /usr/bin/docker && \
-    yum -y remove openssl && \
+    yum -y install java sudo && \
```

Comment:
why do we need `sudo`?

Reply:
probably this is not used anymore but in that case 
https://github.com/eclipse/che/blob/master/dockerfiles/che/Dockerfile#L10 should be removed as well in main image

Reply:
I thought this is needed to be able to run on OpenShift. No?

Reply:
AFAIK using root is not possible on OpenShift in general. I mostly left it to keep it in line with the non-CentOS dockerfile.